appdatabase/migrations/sql/0019_metrics.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS app_metrics (
+       id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
If you don't need an autoincrement field is best not to use one for perf reasons https://stackoverflow.com/questions/52700893/sqlite-best-practices-about-using-autoincrement  
You can use the implicit `rowid` if you need a unique identified, unless you really need an autoincrement field.

Reply:
SQLITE `rowids` are recycled when data is deleted from the table. And we will, at some point delete data from this table. 

I don't know if it matters though. Can give it more thought. Leaving this open for now.

Reply:
Okay, I looked into it. There is no benefit in uniquely identifying a metric. Dropping `AUTOINCREMENT`.